module-2/newspaper/index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>The Guardian</title>
+    <link rel="stylesheet" href="styles.css">
+    <style>
+        html, body {
+            height: 110%;
+        }
+        body {
+            display: flex;
+            flex-direction: column;
+            align-items: center;
+            justify-content: center;
+            margin: 0;
+            overflow-y: auto;
+            background-color: rgb(237, 237, 237);
+        }
+        header {
+            display: flex;
+            flex-direction: row;
+            justify-content: flex-end;
+            width: 100%;
+            padding: 20px;
+        }
+        .side-header {
+            display: flex;
+            flex-direction: column;
+            justify-content: start;
+            width: 100%;
+            padding: 20px;
+        }
+     
+        .main-content {
+            display: flex;
+            flex-direction: row;
+            align-items: center;
+            justify-content: center;
+            gap: 20px;
+            margin: 20px;
+            
+        }
+        .main-content2 {
+            display: flex;
+            flex-direction: row;
+            align-items: center;
+            justify-content: center;
+            gap: 20px;
+            margin: 20px;
+            
+        }
+        footer {
+            width: 100%;
+            text-align: center;
+            padding: 10px;
+            background-color: lightgray;
+        }
+        .row img {
+            width: auto;;
+            height: auto;
+
+        }
+    
+    </style>
+</head>
+<header>
+    <h1 style="text-align: right;">The Guardian</h1>
+    
+</header>
+<body>
+    
+     <h> News | Opinion | Sport| Culture| Lifestyle| More</h>
```

Comment:
Since each of these items (News, Opinions, etc) will be a separate clickable link, you'd want to avoid making them all in one h1 - each could be their own element. Moreover, `<h>` is not a valid HTML element. I think you meant to put `<h1>`

Reply:
Since these items are all links, putting each into their own `<a>` tag would have been a good choice